src/java.desktop/share/classes/sun/awt/OSInfo.java

```diff
@@ -76,7 +82,7 @@ and so the method getWindowsVersion() will return the constant for known OS.
 
     private static final PrivilegedAction<OSType> osTypeAction = new PrivilegedAction<OSType>() {
```

Comment:
As this field is not used in production code anymore, I think it makes sense to inline it (allocate directly in `sun.awt.OSInfo#getOSTypeAction`)